test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -168,6 +170,13 @@ protected override void Cleanup()
                 _normal ? "completed normally" : "aborted",
                 IsPassing ? "passed" : "failed");
 
+            if (_usedSpecificBaselineConfigs.Count > 0)
+            {
+                Log(String.Format("Test {0} is using {1} configuration specific baselines.", TestName,
+                    String.Join(",", _usedSpecificBaselineConfigs)));
```

Comment:
>_usedSpecificBaselineConfigs [](start = 37, length = 28)

If this _usedSpecificBaselineConfigs is for better logging then I'm fine with the change